tests/e2e/high_availability/distributed_coords.py

```diff
@@ -109,22 +110,33 @@
             "--raft-server-port=10113",
         ],
         "log_file": "coordinator3.log",
-        "setup_queries": [
-            "ADD COORDINATOR 1 ON '127.0.0.1:10111'",
-            "ADD COORDINATOR 2 ON '127.0.0.1:10112'",
-            "REGISTER INSTANCE instance_1 ON '127.0.0.1:10011' WITH '127.0.0.1:10001'",
-            "REGISTER INSTANCE instance_2 ON '127.0.0.1:10012' WITH '127.0.0.1:10002'",
-            "REGISTER INSTANCE instance_3 ON '127.0.0.1:10013' WITH '127.0.0.1:10003'",
-            "SET INSTANCE instance_3 TO MAIN",
-        ],
+        "setup_queries": [],
     },
 }
 
 
+def execute_setup_queries(cursor, setup_queries):
+    time.sleep(1)
```

Comment:
Can you adapt PR, I fixed the bug here in #1792. We don't need sleep and all other splitting of setup queries